plugins/env/app/assets/javascripts/env/application.js

```diff
@@ -85,6 +85,31 @@
     return rows;
   }
 
+  // let user paste .env files and we parse out everything
+  function parse(pasted) {
+    if (!pasted) { return; }
```

Comment:
it will never be null ... also empty should be valid, so just remove this line ?